### ISSUE TITLE
libvirt test on CS8 cannot ssh into the VM

### ISSUE THREAD

**Issue by ondrejbudai** [osbuild/osbuild-composer#1454]
No idea what's causing this, see the example:

https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1330953776

**Comment by ondrejbudai** [comment]
Log from a CS8 UEFI VM:

```
[    0.000000] Linux version 4.18.0-277.el8.x86_64 (mockbuild@kbuilder.bsys.centos.org) (gcc version 8.4.1 20200928 (Red Hat 8.4.1-1) (GCC)) #1 SMP Wed Feb 3 20:35:19 UTC 2021
[    0.000000] Command line: BOOT_IMAGE=(hd0,gpt3)/boot/vmlinuz-4.18.0-277.el8.x86_64 root=UUID=5028726e-075f-4504-b42f-26550789f4bc console=tty0 console=ttyS0,115200n8 no_timer_check net.ifnames=0 crashkernel=auto
[    0.000000] x86/fpu: x87 FPU will use FXSAVE
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009ffff] usable
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x000000003e8edfff] usable
[    0.000000] BIOS-e820: [mem 0x000000003e8ee000-0x000000003eaedfff] reserved
[    0.000000] BIOS-e820: [mem 0x000000003eaee000-0x000000003eb75fff] usable
[    0.000000] BIOS-e820: [mem 0x000000003eb76000-0x000000003eb7dfff] ACPI data
[    0.000000] BIOS-e820: [mem 0x000000003eb7e000-0x000000003ebfdfff] ACPI NVS
[    0.000000] BIOS-e820: [mem 0x000000003ebfe000-0x000000003effffff] usable
[    0.000000] BIOS-e820: [mem 0x000000003f000000-0x000000003fffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000b0000000-0x00000000bfffffff] reserved
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] extended physical RAM map:
[    0.000000] reserve setup_data: [mem 0x0000000000000000-0x000000000009ffff] usable
[    0.000000] reserve setup_data: [mem 0x0000000000100000-0x000000003d048017] usable
[    0.000000] reserve setup_data: [mem 0x000000003d048018-0x000000003d072857] usable
[    0.000000] reserve setup_data: [mem 0x000000003d072858-0x000000003d5e4017] usable
[    0.000000] reserve setup_data: [mem 0x000000003d5e4018-0x000000003d5eda57] usable
[    0.000000] reserve setup_data: [mem 0x000000003d5eda58-0x000000003e8edfff] usable
[    0.000000] reserve setup_data: [mem 0x000000003e8ee000-0x000000003eaedfff] reserved
[    0.000000] reserve setup_data: [mem 0x000000003eaee000-0x000000003eb75fff] usable
[    0.000000] reserve setup_data: [mem 0x000000003eb76000-0x000000003eb7dfff] ACPI data
[    0.000000] reserve setup_data: [mem 0x000000003eb7e000-0x000000003ebfdfff] ACPI NVS
[    0.000000] reserve setup_data: [mem 0x000000003ebfe000-0x000000003effffff] usable
[    0.000000] reserve setup_data: [mem 0x000000003f000000-0x000000003fffffff] reserved
[    0.000000] reserve setup_data: [mem 0x00000000b0000000-0x00000000bfffffff] reserved
[    0.000000] efi: EFI v2.70 by EDK II
[    0.000000] efi:  SMBIOS=0x3e9d2000  ACPI=0x3eb7d000  ACPI 2.0=0x3eb7d014  MEMATTR=0x3d5f1518  MOKvar=0x3e974000 
[    0.000000] secureboot: Secure boot disabled
[    0.000000] SMBIOS 2.8 present.
[    0.000000] DMI: Red Hat KVM/RHEL-AV, BIOS 0.0.0 02/06/2015
[    0.000000] last_pfn = 0x3f000 max_arch_pfn = 0x400000000
[    0.000000] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000000] kexec: Reserving the low 1M of memory for crashkernel
[    0.000000] RAMDISK: [mem 0x24c70000-0x288befff]
[    0.000000] ACPI: Early table checksum verification disabled
[    0.000000] ACPI: RSDP 0x000000003EB7D014 000024 (v02 BOCHS )
[    0.000000] ACPI: XSDT 0x000000003EB7C0E8 000044 (v01 BOCHS  BXPCFACP 00000001      01000013)
[    0.000000] ACPI: FACP 0x000000003EB79000 0000F4 (v03 BOCHS  BXPCFACP 00000001 BXPC 00000001)
[    0.000000] ACPI: DSDT 0x000000003EB7A000 001F22 (v01 BOCHS  BXPCDSDT 00000001 BXPC 00000001)
[    0.000000] ACPI: FACS 0x000000003EBDC000 000040
[    0.000000] ACPI: APIC 0x000000003EB78000 000080 (v01 BOCHS  BXPCAPIC 00000001 BXPC 00000001)
[    0.000000] ACPI: MCFG 0x000000003EB77000 00003C (v01 BOCHS  BXPCMCFG 00000001 BXPC 00000001)
[    0.000000] ACPI: BGRT 0x000000003EB76000 000038 (v01 INTEL  EDK2     00000002      01000013)
[    0.000000] No NUMA configuration found
[    0.000000] Faking a node at [mem 0x0000000000000000-0x000000003effffff]
[    0.000000] NODE_DATA(0) allocated [mem 0x3ef4d000-0x3ef76fff]
[    0.000000] Using crashkernel=auto, the size chosen is a best effort estimation.
[    0.000000] Reserving 160MB of memory at 768MB for crashkernel (System RAM: 1005MB)
[    0.000000] Zone ranges:
[    0.000000]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.000000]   DMA32    [mem 0x0000000001000000-0x000000003effffff]
[    0.000000]   Normal   empty
[    0.000000]   Device   empty
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x0000000000001000-0x000000000009ffff]
[    0.000000]   node   0: [mem 0x0000000000100000-0x000000003e8edfff]
[    0.000000]   node   0: [mem 0x000000003eaee000-0x000000003eb75fff]
[    0.000000]   node   0: [mem 0x000000003ebfe000-0x000000003effffff]
[    0.000000] Zeroed struct page in unavailable ranges: 4841 pages
[    0.000000] Initmem setup node 0 [mem 0x0000000000001000-0x000000003effffff]
[    0.000000] ACPI: PM-Timer IO Port: 0x608
[    0.000000] ACPI: LAPIC_NMI (acpi_id[0xff] dfl dfl lint[0x1])
[    0.000000] IOAPIC[0]: apic_id 0, version 32, address 0xfec00000, GSI 0-23
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 high level)
[    0.000000] Using ACPI (MADT) for SMP configuration information
[    0.000000] smpboot: Allowing 2 CPUs, 0 hotplug CPUs
[    0.000000] PM: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.000000] PM: Registered nosave memory: [mem 0x000a0000-0x000fffff]
[    0.000000] PM: Registered nosave memory: [mem 0x3d048000-0x3d048fff]
[    0.000000] PM: Registered nosave memory: [mem 0x3d072000-0x3d072fff]
[    0.000000] PM: Registered nosave memory: [mem 0x3d122000-0x3d148fff]
[    0.000000] PM: Registered nosave memory: [mem 0x3d5e4000-0x3d5e4fff]
[    0.000000] PM: Registered nosave memory: [mem 0x3d5ed000-0x3d5edfff]
[    0.000000] PM: Registered nosave memory: [mem 0x3e8ee000-0x3eaedfff]
[    0.000000] PM: Registered nosave memory: [mem 0x3eb76000-0x3eb7dfff]
[    0.000000] PM: Registered nosave memory: [mem 0x3eb7e000-0x3ebfdfff]
[    0.000000] [mem 0x40000000-0xafffffff] available for PCI devices
[    0.000000] Booting paravirtualized kernel on bare hardware
[    0.000000] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 1910969940391419 ns
[    0.000000] setup_percpu: NR_CPUS:8192 nr_cpumask_bits:2 nr_cpu_ids:2 nr_node_ids:1
[    0.000000] percpu: Embedded 54 pages/cpu s184320 r8192 d28672 u1048576
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 251096
[    0.000000] Policy zone: DMA32
[    0.000000] Kernel command line: BOOT_IMAGE=(hd0,gpt3)/boot/vmlinuz-4.18.0-277.el8.x86_64 root=UUID=5028726e-075f-4504-b42f-26550789f4bc console=tty0 console=ttyS0,115200n8 no_timer_check net.ifnames=0 crashkernel=auto
[    0.000000] Specific versions of hardware are certified with Red Hat Enterprise Linux 8. Please see the list of hardware certified with Red Hat Enterprise Linux 8 at https://catalog.redhat.com.
[    0.000000] Memory: 698712K/1029212K available (12293K kernel code, 2216K rwdata, 4052K rodata, 2468K init, 14612K bss, 330500K reserved, 0K cma-reserved)
[    0.000000] random: get_random_u64 called from cache_random_seq_create+0x7c/0x140 with crng_init=0
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
[    0.000000] ftrace: allocating 38271 entries in 150 pages
[    0.000000] ftrace: allocated 150 pages with 4 groups
[    0.000000] rcu: Hierarchical RCU implementation.
[    0.000000] rcu: 	RCU restricting CPUs from NR_CPUS=8192 to nr_cpu_ids=2.
[    0.000000] rcu: RCU calculated value of scheduler-enlistment delay is 100 jiffies.
[    0.000000] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=2
[    0.000000] NR_IRQS: 524544, nr_irqs: 440, preallocated irqs: 16
[    0.000000] Console: colour dummy device 80x25
[    0.000000] printk: console [tty0] enabled
[    0.000000] printk: console [ttyS0] enabled
[    0.000000] ACPI: Core revision 20200110
[    0.001000] APIC: Switch to symmetric I/O mode setup
[    0.002000] ..TIMER: vector=0x30 apic1=0 pin1=2 apic2=-1 pin2=-1
[    0.003000] tsc: Unable to calibrate against PIT
[    0.003000] tsc: using PMTIMER reference calibration
[    0.003000] tsc: Detected 2999.946 MHz processor
[    0.003000] tsc: Marking TSC unstable due to TSCs unsynchronized
[    0.003000] Calibrating delay loop (skipped), value calculated using timer frequency.. 5999.89 BogoMIPS (lpj=2999946)
[    0.003000] pid_max: default: 32768 minimum: 301
[    0.009000] Security Framework initialized
[    0.009000] Yama: becoming mindful.
[    0.010605] SELinux:  Initializing.
[    0.013000] Dentry cache hash table entries: 131072 (order: 8, 1048576 bytes)
[    0.013612] Inode-cache hash table entries: 65536 (order: 7, 524288 bytes)
[    0.014000] Mount-cache hash table entries: 2048 (order: 2, 16384 bytes)
[    0.014000] Mountpoint-cache hash table entries: 2048 (order: 2, 16384 bytes)
[    0.020000] Last level iTLB entries: 4KB 0, 2MB 0, 4MB 0
[    0.020000] Last level dTLB entries: 4KB 0, 2MB 0, 4MB 0, 1GB 0
[    0.020026] FEATURE SPEC_CTRL Not Present
[    0.020194] FEATURE IBPB_SUPPORT Not Present
[    0.020398] Spectre V1 : Mitigation: usercopy/swapgs barriers and __user pointer sanitization
[    0.020814] Spectre V2 : Spectre mitigation: LFENCE not serializing, switching to generic retpoline
[    0.021000] Spectre V2 : Mitigation: Full generic retpoline
[    0.021000] Spectre V2 : Spectre v2 / SpectreRSB mitigation: Filling RSB on context switch
[    0.021000] Speculative Store Bypass: Vulnerable
[    0.023000] Freeing SMP alternatives memory: 32K
[    0.028000] APIC calibration not consistent with PM-Timer: 1235ms instead of 100ms
[    0.028000] APIC delta adjusted to PM-Timer: 6249994 (77247136)
[    0.029000] smpboot: CPU0: AMD QEMU Virtual CPU version 2.5+ (family: 0x6, model: 0xd, stepping: 0x3)
[    0.030000] Performance Events: PMU not available due to virtualization, using software events only.
[    0.031000] rcu: Hierarchical SRCU implementation.
[    0.032000] NMI watchdog: Perf NMI watchdog permanently disabled
[    0.033000] smp: Bringing up secondary CPUs ...
[    0.034000] x86: Booting SMP configuration:
[    0.034000] .... node  #0, CPUs:      #1
[    0.001000] smpboot: CPU 1 Converting physical 0 to logical die 1
[    0.001000] calibrate_delay_direct() dropping max bogoMips estimate 3 = 37196680
[    0.108127] smp: Brought up 1 node, 2 CPUs
[    0.109064] smpboot: Max logical packages: 2
[    0.109219] smpboot: Total of 2 processors activated (67799.48 BogoMIPS)
[    0.128085] devtmpfs: initialized
[    0.132717] x86/mm: Memory block size: 128MB
[    0.138504] PM: Registering ACPI NVS region [mem 0x3eb7e000-0x3ebfdfff] (524288 bytes)
[    0.140202] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 1911260446275000 ns
[    0.140802] futex hash table entries: 512 (order: 3, 32768 bytes)
[    0.144433] pinctrl core: initialized pinctrl subsystem
[    0.152315] NET: Registered protocol family 16
[    0.155146] audit: initializing netlink subsys (disabled)
[    0.157417] audit: type=2000 audit(1623309361.157:1): state=initialized audit_enabled=0 res=1
[    0.162803] cpuidle: using governor menu
[    0.164767] ACPI: bus type PCI registered
[    0.165000] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    0.167576] PCI: MMCONFIG for domain 0000 [bus 00-ff] at [mem 0xb0000000-0xbfffffff] (base 0xb0000000)
[    0.169233] PCI: MMCONFIG at [mem 0xb0000000-0xbfffffff] reserved in E820
[    0.170047] PCI: Using configuration type 1 for base access
[    0.172362] mtrr: your CPUs had inconsistent fixed MTRR settings
[    0.172587] mtrr: your CPUs had inconsistent variable MTRR settings
[    0.172801] mtrr: your CPUs had inconsistent MTRRdefType settings
[    0.172989] mtrr: probably your BIOS does not setup all CPUs.
[    0.173000] mtrr: corrected configuration.
[    0.190304] HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
[    0.193228] cryptd: max_cpu_qlen set to 1000
[    0.197262] ACPI: Added _OSI(Module Device)
[    0.197449] ACPI: Added _OSI(Processor Device)
[    0.197593] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.197757] ACPI: Added _OSI(Processor Aggregator Device)
[    0.198038] ACPI: Added _OSI(Linux-Dell-Video)
[    0.198189] ACPI: Added _OSI(Linux-Lenovo-NV-HDMI-Audio)
[    0.198358] ACPI: Added _OSI(Linux-HPI-Hybrid-Graphics)
[    0.216078] ACPI: 1 ACPI AML tables successfully acquired and loaded
[    0.230665] ACPI: Interpreter enabled
[    0.231553] ACPI: (supports S0 S5)
[    0.231740] ACPI: Using IOAPIC for interrupt routing
[    0.233195] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.235000] ACPI: Enabled 1 GPEs in block 00 to 3F
[    0.261858] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.262230] acpi PNP0A08:00: _OSC: OS supports [ExtendedConfig ASPM ClockPM Segments MSI EDR HPX-Type3]
[    0.264389] acpi PNP0A08:00: _OSC: platform does not support [LTR DPC]
[    0.265436] acpi PNP0A08:00: _OSC: OS now controls [PCIeHotplug SHPCHotplug PME AER PCIeCapability]
[    0.269677] PCI host bridge to bus 0000:00
[    0.270059] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.270297] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.270511] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    0.270754] pci_bus 0000:00: root bus resource [mem 0x40000000-0xafffffff window]
[    0.270987] pci_bus 0000:00: root bus resource [mem 0xc0000000-0xfebfffff window]
[    0.271000] pci_bus 0000:00: root bus resource [mem 0x800000000-0xfffffffff window]
[    0.271071] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.272545] pci 0000:00:00.0: [8086:29c0] type 00 class 0x060000
[    0.277136] pci 0000:00:01.0: [1b36:0100] type 00 class 0x030000
[    0.279257] pci 0000:00:01.0: reg 0x10: [mem 0xc4000000-0xc7ffffff]
[    0.281000] pci 0000:00:01.0: reg 0x14: [mem 0xc0000000-0xc3ffffff]
[    0.284054] pci 0000:00:01.0: reg 0x18: [mem 0xc8e44000-0xc8e45fff]
[    0.287050] pci 0000:00:01.0: reg 0x1c: [io  0xc260-0xc27f]
[    0.293000] pci 0000:00:01.0: reg 0x30: [mem 0xffff0000-0xffffffff pref]
[    0.293345] pci 0000:00:01.0: BAR 0: assigned to efifb
[    0.294897] pci 0000:00:02.0: [1b36:000c] type 01 class 0x060400
[    0.297888] pci 0000:00:02.0: reg 0x10: [mem 0xc8e4d000-0xc8e4dfff]
[    0.307780] pci 0000:00:02.1: [1b36:000c] type 01 class 0x060400
[    0.310000] pci 0000:00:02.1: reg 0x10: [mem 0xc8e4c000-0xc8e4cfff]
[    0.315336] pci 0000:00:02.2: [1b36:000c] type 01 class 0x060400
[    0.317797] pci 0000:00:02.2: reg 0x10: [mem 0xc8e4b000-0xc8e4bfff]
[    0.323703] pci 0000:00:02.3: [1b36:000c] type 01 class 0x060400
[    0.326000] pci 0000:00:02.3: reg 0x10: [mem 0xc8e4a000-0xc8e4afff]
[    0.331335] pci 0000:00:02.4: [1b36:000c] type 01 class 0x060400
[    0.333828] pci 0000:00:02.4: reg 0x10: [mem 0xc8e49000-0xc8e49fff]
[    0.341205] pci 0000:00:02.5: [1b36:000c] type 01 class 0x060400
[    0.342787] pci 0000:00:02.5: reg 0x10: [mem 0xc8e48000-0xc8e48fff]
[    0.348619] pci 0000:00:02.6: [1b36:000c] type 01 class 0x060400
[    0.350000] pci 0000:00:02.6: reg 0x10: [mem 0xc8e47000-0xc8e47fff]
[    0.355768] pci 0000:00:1b.0: [8086:293e] type 00 class 0x040300
[    0.357478] pci 0000:00:1b.0: reg 0x10: [mem 0xc8e40000-0xc8e43fff]
[    0.363463] pci 0000:00:1f.0: [8086:2918] type 00 class 0x060100
[    0.364199] pci 0000:00:1f.0: quirk: [io  0x0600-0x067f] claimed by ICH6 ACPI/GPIO/TCO
[    0.365173] pci 0000:00:1f.2: [8086:2922] type 00 class 0x010601
[    0.372069] pci 0000:00:1f.2: reg 0x20: [io  0xc240-0xc25f]
[    0.373771] pci 0000:00:1f.2: reg 0x24: [mem 0xc8e46000-0xc8e46fff]
[    0.377114] pci 0000:00:1f.3: [8086:2930] type 00 class 0x0c0500
[    0.382000] pci 0000:00:1f.3: reg 0x20: [io  0xc200-0xc23f]
[    0.387187] pci 0000:01:00.0: [1af4:1041] type 00 class 0x020000
[    0.390070] pci 0000:01:00.0: reg 0x14: [mem 0xc8c00000-0xc8c00fff]
[    0.396063] pci 0000:01:00.0: reg 0x20: [mem 0x800000000-0x800003fff 64bit pref]
[    0.398040] pci 0000:01:00.0: reg 0x30: [mem 0xfffc0000-0xffffffff pref]
[    0.400908] pci 0000:00:02.0: PCI bridge to [bus 01]
[    0.401042] pci 0000:00:02.0:   bridge window [io  0xc000-0xcfff]
[    0.401331] pci 0000:00:02.0:   bridge window [mem 0xc8c00000-0xc8dfffff]
[    0.401721] pci 0000:00:02.0:   bridge window [mem 0x800000000-0x8000fffff 64bit pref]
[    0.404426] pci 0000:02:00.0: [1b36:000d] type 00 class 0x0c0330
[    0.406045] pci 0000:02:00.0: reg 0x10: [mem 0xc8a00000-0xc8a03fff 64bit]
[    0.410554] pci 0000:00:02.1: PCI bridge to [bus 02]
[    0.410780] pci 0000:00:02.1:   bridge window [io  0xb000-0xbfff]
[    0.410986] pci 0000:00:02.1:   bridge window [mem 0xc8a00000-0xc8bfffff]
[    0.412277] pci 0000:03:00.0: [1af4:1043] type 00 class 0x078000
[    0.417055] pci 0000:03:00.0: reg 0x14: [mem 0xc8800000-0xc8800fff]
[    0.422048] pci 0000:03:00.0: reg 0x20: [mem 0x800100000-0x800103fff 64bit pref]
[    0.424640] pci 0000:00:02.2: PCI bridge to [bus 03]
[    0.425031] pci 0000:00:02.2:   bridge window [io  0xa000-0xafff]
[    0.425148] pci 0000:00:02.2:   bridge window [mem 0xc8800000-0xc89fffff]
[    0.425270] pci 0000:00:02.2:   bridge window [mem 0x800100000-0x8001fffff 64bit pref]
[    0.426819] pci 0000:04:00.0: [1af4:1042] type 00 class 0x010000
[    0.430776] pci 0000:04:00.0: reg 0x14: [mem 0xc8600000-0xc8600fff]
[    0.434780] pci 0000:04:00.0: reg 0x20: [mem 0x800200000-0x800203fff 64bit pref]
[    0.437000] pci 0000:00:02.3: PCI bridge to [bus 04]
[    0.437000] pci 0000:00:02.3:   bridge window [io  0x9000-0x9fff]
[    0.437000] pci 0000:00:02.3:   bridge window [mem 0xc8600000-0xc87fffff]
[    0.437000] pci 0000:00:02.3:   bridge window [mem 0x800200000-0x8002fffff 64bit pref]
[    0.439346] pci 0000:05:00.0: [1af4:1045] type 00 class 0x00ff00
[    0.444053] pci 0000:05:00.0: reg 0x20: [mem 0x800300000-0x800303fff 64bit pref]
[    0.446149] pci 0000:00:02.4: PCI bridge to [bus 05]
[    0.446359] pci 0000:00:02.4:   bridge window [io  0x8000-0x8fff]
[    0.446576] pci 0000:00:02.4:   bridge window [mem 0xc8400000-0xc85fffff]
[    0.446824] pci 0000:00:02.4:   bridge window [mem 0x800300000-0x8003fffff 64bit pref]
[    0.448320] pci 0000:06:00.0: [1af4:1044] type 00 class 0x00ff00
[    0.454040] pci 0000:06:00.0: reg 0x20: [mem 0x800400000-0x800403fff 64bit pref]
[    0.456193] pci 0000:00:02.5: PCI bridge to [bus 06]
[    0.456417] pci 0000:00:02.5:   bridge window [io  0x7000-0x7fff]
[    0.456648] pci 0000:00:02.5:   bridge window [mem 0xc8200000-0xc83fffff]
[    0.456897] pci 0000:00:02.5:   bridge window [mem 0x800400000-0x8004fffff 64bit pref]
[    0.458262] pci 0000:00:02.6: PCI bridge to [bus 07]
[    0.458483] pci 0000:00:02.6:   bridge window [io  0x6000-0x6fff]
[    0.458688] pci 0000:00:02.6:   bridge window [mem 0xc8000000-0xc81fffff]
[    0.472374] ACPI: PCI Interrupt Link [LNKA] (IRQs 5 *10 11)
[    0.474383] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    0.475155] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    0.475833] ACPI: PCI Interrupt Link [LNKD] (IRQs 5 10 *11)
[    0.476416] ACPI: PCI Interrupt Link [LNKE] (IRQs 5 *10 11)
[    0.477332] ACPI: PCI Interrupt Link [LNKF] (IRQs 5 *10 11)
[    0.478060] ACPI: PCI Interrupt Link [LNKG] (IRQs 5 10 *11)
[    0.478700] ACPI: PCI Interrupt Link [LNKH] (IRQs 5 10 *11)
[    0.479132] ACPI: PCI Interrupt Link [GSIA] (IRQs *16)
[    0.479412] ACPI: PCI Interrupt Link [GSIB] (IRQs *17)
[    0.479730] ACPI: PCI Interrupt Link [GSIC] (IRQs *18)
[    0.480000] ACPI: PCI Interrupt Link [GSID] (IRQs *19)
[    0.480290] ACPI: PCI Interrupt Link [GSIE] (IRQs *20)
[    0.480607] ACPI: PCI Interrupt Link [GSIF] (IRQs *21)
[    0.481019] ACPI: PCI Interrupt Link [GSIG] (IRQs *22)
[    0.481314] ACPI: PCI Interrupt Link [GSIH] (IRQs *23)
[    0.485176] iommu: Default domain type: Passthrough 
[    0.489467] pci 0000:00:01.0: vgaarb: setting as boot VGA device
[    0.489831] pci 0000:00:01.0: vgaarb: VGA device added: decodes=io+mem,owns=io+mem,locks=none
[    0.490000] pci 0000:00:01.0: vgaarb: bridge control possible
[    0.490191] vgaarb: loaded
[    0.492595] SCSI subsystem initialized
[    0.493412] ACPI: bus type USB registered
[    0.494151] usbcore: registered new interface driver usbfs
[    0.494700] usbcore: registered new interface driver hub
[    0.495269] usbcore: registered new device driver usb
[    0.496073] pps_core: LinuxPPS API ver. 1 registered
[    0.496310] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.496719] PTP clock support registered
[    0.498104] EDAC MC: Ver: 3.0.0
[    0.499500] Registered efivars operations
[    0.503000] PCI: Using ACPI for IRQ routing
[    0.504000] pci 0000:00:01.0: can't claim BAR 3 [io  0xc260-0xc27f]: address conflict with PCI Bus 0000:01 [io  0xc000-0xcfff]
[    0.504000] pci 0000:00:1f.2: can't claim BAR 4 [io  0xc240-0xc25f]: address conflict with PCI Bus 0000:01 [io  0xc000-0xcfff]
[    0.504000] pci 0000:00:1f.3: can't claim BAR 4 [io  0xc200-0xc23f]: address conflict with PCI Bus 0000:01 [io  0xc000-0xcfff]
[    0.504000] NetLabel: Initializing
[    0.504000] NetLabel:  domain hash size = 128
[    0.504000] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.504000] NetLabel:  unlabeled traffic allowed by default
```

Interestingly enough, I'm able to boot the cs8 image built by osbuild-composer on Fedora just fine. Both with and without KVM.

**Comment by ondrejbudai** [comment]
Fixed by #1459
